src/main/java/org/xstefank/check/SkipCheck.java

```diff
@@ -1,41 +1,50 @@
 package org.xstefank.check;
 
 import com.fasterxml.jackson.databind.JsonNode;
+import org.jboss.logging.Logger;
+import org.xstefank.api.GitHubAPI;
 import org.xstefank.model.Utils;
 import org.xstefank.model.yaml.FormatConfig;
 
 import java.util.regex.Matcher;
 
 public class SkipCheck {
 
+    private static final Logger log = Logger.getLogger(SkipCheck.class);
+
     public static boolean shouldSkip(JsonNode payload, FormatConfig config) {
-        if (config == null) {
-            throw new IllegalArgumentException("Argument config is null!");
+        if (payload == null || config == null) {
+            throw new IllegalArgumentException("Argument payload or config is null!");
         }
-        return skipByTitle(payload, config) || skipByCommit(payload, config) || skipByDescriptionFirstRow(payload, config);
+        log.info("checking skipPatterns");
+        return skipByTitle(payload, config) || skipByCommit(payload, config) || skipByPullRequestDescriptionFirstRow(payload, config);
     }
 
     private static boolean skipByTitle(JsonNode payload, FormatConfig config) {
-        if (config.getFormat().getSkipPatterns().getTitle() != null) {
-            Matcher titleMatcher = config.getFormat().getSkipPatterns().getTitle().matcher(payload.get(Utils.PULL_REQUEST).get(Utils.TITLE).asText());
+        if (config.getFormat().getSkipPatterns().getTitlePattern() != null) {
+            log.info("checking title");
+            Matcher titleMatcher = config.getFormat().getSkipPatterns().getTitlePattern().matcher(payload.get(Utils.PULL_REQUEST).get(Utils.TITLE).asText());
             return titleMatcher.matches();
         }
         return false;
     }
 
     private static boolean skipByCommit(JsonNode payload, FormatConfig config) {
-        if (config.getFormat().getSkipPatterns().getCommit() != null) {
-            LatestCommitCheck latestCommitCheck = new LatestCommitCheck(config.getFormat().getSkipPatterns().getCommit());
-            return (latestCommitCheck.check(payload) == null);
```

Comment:
does it make sense to have separate patterns for latestCommitCheck and skipPatterns commit pattern?